src/test/java/ProjectileTest.java

```diff
@@ -0,0 +1,33 @@
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class ProjectileTest
+{
+    @Test
+    public void getX()
+    {
+        // given - variables, conditions
+        Projectile p = new Projectile(52, 17);
+
+        // when - tests
+        double x = p.getX(3);   // 31.3987
```

Comment:
No need to put the number as a comment here.
I did it in class to save it before I put in the assert